### PR TITLE
docs: add how-to for cert management

### PR DESCRIPTION
**Pull request by steverfrancis**
# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

**Comment by nberlee** [comment]
Default kubeconfig is 1 year, but the doc is incorrect about `talosctl config new` which generates *10 years* by default:
```
$ talosctl config new --help
  talosctl config new [<path>] [flags]

Flags:
      --crt-ttl duration   certificate TTL (default 87600h0m0s)
```

**Comment by smira** [comment]
> Default kubeconfig is 1 year, but the doc is incorrect about `talosctl config new` which generates _10 years_ by default:
> Flags:
>       --crt-ttl duration   certificate TTL (default 87600h0m0s)
> ```

This feels like a bug actually, I will take a look

File: website/content/v1.6/talos-guides/howto/cert-management.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "How to manage certificate lifetimess with Talos Linux"
```
Comment (smira) [inline website/content/v1.6/talos-guides/howto/cert-management.md:2]:
```suggestion
title: "How to manage certificate lifetimes with Talos Linux"
```

?

Comment (smira) [reply website/content/v1.6/talos-guides/howto/cert-management.md:2]: Or should it be singular?

**Pull request merged**